src/android/SplashScreen.java

```diff
@@ -254,8 +262,12 @@ public void run() {
                 }
                 splashDialog.setContentView(splashImageView);
                 splashDialog.setCancelable(false);
-                splashDialog.show();
-
+                if (isHideNavigationBar() || isHideStatusBar()) {
+					splashDialog.getWindow().getDecorView().setSystemUiVisibility(
+					(isHideNavigationBar() ? View.SYSTEM_UI_FLAG_HIDE_NAVIGATION : 0) | (isHideNavigationBar() ? View.SYSTEM_UI_FLAG_FULLSCREEN : 0));
```

Comment:
The indenting on pretty much every line you changed looks off. Likely the problem is that your editor is inserting tabs instead of spaces.